recipe/test/CMakeLists.txt

```
@@ -1,13 +1,30 @@
-cmake_minimum_required (VERSION 3.0)
+cmake_minimum_required(VERSION 3.16)
 
-set (CMAKE_BUILD_TYPE "Release" CACHE STRING "build type")
+project(qtsvg-test CXX)
 
-project (qtsvg-test CXX)
+# Set C++ standard
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-find_package (Qt6 CONFIG REQUIRED COMPONENTS Svg)
+# Try to find Qt6 components directly first, fallback to main package
+find_package(Qt6Core QUIET)
+find_package(Qt6Svg QUIET)
 
-add_executable (test_qtsvg test_qtsvg.cpp)
-target_link_libraries (test_qtsvg Qt6::Svg)
+if(Qt6Core_FOUND AND Qt6Svg_FOUND)
+    # Direct component approach worked
+    add_executable(test_qtsvg test_qtsvg.cpp)
+    target_link_libraries(test_qtsvg Qt6::Core Qt6::Svg)
+else()
+    # Fallback to main Qt6 package
+    find_package(Qt6 REQUIRED COMPONENTS Core Svg)
+    add_executable(test_qtsvg test_qtsvg.cpp)
+    target_link_libraries(test_qtsvg Qt6::Core Qt6::Svg)
+endif()
```

```suggestion
find_package (Qt6 CONFIG REQUIRED COMPONENTS Core Svg)

add_executable(test_qtsvg test_qtsvg.cpp)
target_link_libraries(test_qtsvg Qt6::Core Qt6::Svg)
```
There really is no "fallback" method here since the test links against both QtCore and QtSvg. Both need to be present. The only difference between finding them as components versus finding them with their own CMake config files is this way uses a parent `Qt6Config.cmake` file to dispatch the searches instead of calling directly into `Qt6CoreConfig.cmake` and `Qt6SvgConfig.cmake`. They both eventually use the same files to locate the libs and include dirs.

Since `Qt6Config.cmake` comes with `qtbase-devel` and that is also where QtCore resides, it is not possible to be able to run the tests without also having `find_package(Qt6 COMPONENTS Core Svg)` working properly. Additionally, this is how [their docs](https://doc.qt.io/qt-6/cmake-get-started.html) describe finding Qt modules and is likely how our customers would as well.

---

Done